src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -294,7 +296,11 @@ describe('ReactDOMComponent', function() {
       var container = document.createElement('div');
       ReactDOM.render(<svg theWord="theBird" />, container);
 
-      expect(container.firstChild.hasAttribute('theWord')).toBe(true);
+      if (ReactDOMFeatureFlags.useCreateElement) {
+        // jsdom's svg parsing is a little buggy and makes attributes lowercase,
+        // only assert this in createElement mode...
+        expect(container.firstChild.hasAttribute('theWord')).toBe(true);
+      }
```

Comment:
cc @cpojer - is there a jsdom update we need to apply to jest or can we look into fixing it?